core/server/translations/en.json

```diff
@@ -77,7 +77,14 @@
                 "mismatchedOrigin": "Request made from incorrect origin.",
                 "missingUserIDForSession": "Cannot create session without user id.",
                 "accessDenied": "Access denied.",
-                "pleaseSignIn": "Please Sign In"
+                "pleaseSignIn": "Please Sign In",
+                "pleaseSignInOrAuthenticate": "Please sign in or authenticate with an API Key",
+                "unknownAdminApiKey": "Unknown Admin API Key",
+                "unknownContentApiKey": "Unknown Content API Key",
+                "invalidApiKeyType": "Invalid API Key type",
+                "invalidJwt": "Invalid JWT",
```

Comment:
> Invalid JWT

We could make the messages using "JWT" a little bit more general.

Reply:
Hmm, yeah - though these are specifically for malformed/invalid JWT, which would help a developer when not using the SDK. Can just change to "Invalid token"?

Reply:
👍